Documentation/applications/interpreters/wamr/index.rst

```diff
@@ -1,3 +1,23 @@
 ==================================
 ``wamr`` Webassembly Micro Runtime
 ==================================
+
+This captures how to try WAMR on NuttX.
```

Comment:
This guides explains how to test WAMR on NuttX and how to get it working correctly.

Reply:
updated